clients/fides-js/src/lib/i18n/locales/ar/messages.json

```diff
@@ -1,18 +1,19 @@
 {
-  "static.tcf.purposes": "الأغراض",
```

Comment:
You can mostly ignore all the changes in these messages.json files - I decided to go ahead and sort the JSON keys alphabetically so that future updates would be easier to grok.